unit-tests/shared/src/test/scala/org/scalanative/testsuite/javalib/net/DatagramSocketTest.scala

```diff
@@ -395,6 +395,26 @@ class DatagramSocketTest {
   }
 
   @Test def sendReceiveBroadcast(): Unit = {
+    /* Issue 4221
+     *   This Test should be run only in environments where:
+     *   1) It is responsible to broadcast packets to every node on net.
+     *   2) The network configuration, possible firewalls, and routers
+     *      allow broadcasting.
+     *
+     *   This is true in the Scala Native Continuous Integration environment
+     *   but may not be in more general work environments.
+     *
+     *   Network developers will need to bypass this check and run the
+     *   test manually.
+     */
+    assumeTrue(
+      "Advanced and/or CI-only test",
+      ju.Optional
+        .ofNullable(System.getenv("GITHUB_ACTIONS"))
+        .orElse("false")
+        .equalsIgnoreCase("true")
```

Comment:
It can be safely simplified to just 
```suggestion
      sys.env.contains("CI")
```
Similar approach was used in other tests

Reply:
Thank you for the suggestion.
> Similar approach was used in other tests

I've learned since then.  In a development environment, one has to go out of their way
to define `GITHUB_ACTIONS`. It is perfectly reasonable for a user to have defined `CI`
for something pertinent to their own environment.  Perhaps the devo is one of many and
is named "Charles Ingalls"?

I can make the change next week if you like the `CI` approach.

If you/we decide to go the `GITHUB_ACTIONS` route,  for consistency I can go back and
change the other occurrences of `CI` to the former. I am probably the only one who has
ever used that path.